common/src/main/java/com/genexus/xml/XMLReader.java

```diff
@@ -882,8 +884,9 @@ public void close()
 						is.close();
 					}
 				}
+				if (inputSource.getByteStream() != null) inputSource.getByteStream().close();
 			}
-			parserConfiguration.cleanup();			
```

Comment:
It would be nice to add the "finalize" Java Standard method so when the GarbageCollector tries to delete the object, we are sure that the Stream is closed, in case a GeneXus Developer did't call the &XmlReader.Close()


In this case, we may need to check in the Close() MEthod if parserConfiguration is != NULL before calling "Cleanup".